compiler/x/codegen/OMRInstruction.hpp

```diff
@@ -284,8 +284,75 @@ class OMR_EXTENSIBLE Instruction : public OMR::Instruction
       {
       VEX() {TR_ASSERT(false, "INVALID VEX PREFIX");}
       };
+   template<size_t EVEX_SIZE>
+   struct EVEX
+      {
+         EVEX() {TR_ASSERT(false, "INVALID EVEX PREFIX");}
+      };
    };
 
+   template<>
+   struct Instruction::EVEX<4>
+      {
+      // Byte 0: 0x62
+      uint8_t escape;
+      // Byte 1
+      uint8_t m : 2;
+      uint8_t zero : 2; // 0b00
+      uint8_t r : 1;
```

Comment:
Can you add documentation that ties each of this bits back to the prefix bit as described in the Intel manual?  e.g., 

`// P[4]  : R'`

It makes it easier to relate what these bits represent as well as what their offsets are (without doing the math each time).